src/components/FlowListItem.vue

```diff
@@ -156,12 +155,14 @@
 
 .flow-list-item__content-toggle { @apply
   relative
-  rounded-full
-  transition-transform
   mr-2
 }
 
-.flow-list-item__content-toggle--expanded { @apply
+.flow-list-item__content-toggle > * { @apply
+  transition-transform
+}
+
+.flow-list-item__content-toggle--expanded > * { @apply
```

Comment:
Curious why this selector needed to be updated like this. I'd like to avoid using `*` if possible. Also the transition should be inherited right?

Reply:
Yeah just trying to target the icon in a sustainable way.. I should probably just target `.p-icon`, huh? That way it's all good if extra containers get added in p-button or something

Reply:
Yeah that would be better. This is one case where having a slot for the icon might be preferred so that at implementation the icon can have its own class. I like the devx of the icon props. But having slots for them might be nice for stuff like this. 

For now updating this to target the `p-icon` itself would be an improvement 👍